Documentation/guides/messages/xa0125.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Xamarin.Android warning XA0125
+description: XA0125 warning code
+ms.date: 09/28/2020
+---
+# Xamarin.Android warning XA0125
+
```

Comment:
To make this page potentially more directly discoverable via web search for anyone hitting the message, an _Example messages_ section could optionally be added:

````markdown
## Example messages

```
warning XA0125: 'AndroidApp1.pdb' is using a deprecated debug information level. Set the debugging information to Portable in the Visual Studio project property pages or edit the project file in a text editor and set the 'DebugType' MSBuild property to 'portable' to use the newer, cross-platform debug information level.
```
````

(The example text here matches the possible wording ideas mentioned in another comment in this PR review for the `.resx` file.)